mariadb.html.md.erb

```diff
@@ -47,4 +47,19 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-The base of our HA MariaDB offering is built by a galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+The base of our HA MariaDB offering is built by a Galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+
+Our Galera aaS is a shared Galera cluster. Each customer have their own Database (`CREATE DATABSE`) with rights inside their own database.
```

Comment:
* should be "each customer has their own" instead of "each customer have their own"
* the sentence doesn't really make sense... they have "their own Database with rights inside their own database"??